armi/reactor/converters/blockConverters.py

```diff
@@ -824,3 +862,61 @@ def radiiFromRingOfRods(distToRodCenter, numRods, rodRadii, layout="hexagon"):
         rLast, bigRLast = rodRadius, distFromCenterComp
 
     return sorted(radiiFromRodCenter)
+
+
+def stripComponents(block, compFlags):
```

Comment:
Would a more accurate neme be `mergeInnerComponents`?

Reply:
This function doesn't actually homogenize the components, it just creates a new block and pulls off the components that aren't going to be homogenized, and then returns that block.